base/applications/utilman/dialog.c

```diff
@@ -89,20 +89,26 @@ BOOL DlgInitHandler(HWND hDlg)
     /* Refresh the list */
     ListBoxRefreshContents();
 
-    /* Load the groupbox's title string */
-    LoadStringW(Globals.hInstance, IDC_GROUPBOX_OPTIONS_TITLE, EntriesList[0].szCaptionUtility, _countof(EntriesList[0].szCaptionUtility));
-
-    /*
-        Check that the characters are actually copied into the buffer, then append
-        the string using StringCchPrintfW() to the format.
-    */
-    if (LoadStringW(Globals.hInstance, IDC_MAGNIFIER, Globals.szResource, _countof(Globals.szResource)))
+    for (Globals.iArrayIndex = 0; Globals.iArrayIndex < _countof(EntriesList); Globals.iArrayIndex++)
     {
-        StringCchPrintfW(Globals.szFormat, _countof(Globals.szFormat), EntriesList[0].szCaptionUtility, Globals.szResource);
-    }
+        if (Globals.iArrayIndex == 0)
+        {
+            /* Load the groupbox's title string */
+            LoadStringW(Globals.hInstance, IDC_GROUPBOX_OPTIONS_TITLE, EntriesList[Globals.iArrayIndex].szCaptionUtility, _countof(EntriesList[Globals.iArrayIndex].szCaptionUtility));
+
+            /*
+                Check that the characters are actually copied into the buffer, then append
```

Comment:
These kind of comments are useless in the code since they just paraphrase what the code litterally does below.
Good comments are those that explain what the purpose of an algorithm is, describing the line of thought, etc., but not stuff like:
```c
/* Return status */
return Status;
```
and the like as done so many times in our code base.